### PR TITLE
Adds another hardsuit reaction message

### PR DESCRIPTION
**Pull request by yoyobatty**
[Guidelines]: # (Be sure that your PR follows our guidelines, such as modularization and comment standards. You can read more about the subject here: https://github.com/HippieStation/HippieStation/blob/master/hippiestation/README.md )

## Changelog
:cl: YoYoBatty
add: Hardsuits now react when being shot with a message.
tweak: Cleaned up hardsuit reaction messages and unified the cooldown timer for all messages.
/:cl:

## About The Pull Request
This basically makes hardsuits have a reaction when getting shot. Like other messages, has a cooldown. I also took the time to cleanup up some of the code I wrote a long time ago and put all the messages under one single cool down. These messages don't happen that often and I don't want to confuse the user with a bunch of spam.

## Why It's Good For The Game
It's a neat little hint that adds a bit of little bit more personality to hardsuits.

### CODE REVIEW

File: hippiestation/code/modules/clothing/spacesuits/hardsuit.dm [hunk 1]
```diff
@@ -1,27 +1,37 @@
...
+	if(prob(33) && acidpwr >= 10 && helmet)
+		if(helmet.next_warn > world.time)
+			return
+		helmet.next_warn = world.time + WARN_COOLDOWN_TIMER
+		helmet.display_visor_message("Corrosive chemical detected!")
+
+/obj/item/clothing/suit/space/hardsuit/hit_reaction(mob/living/carbon/human/user, atom/movable/hitby, attack_text = "the attack", final_block_chance = 0, damage = 0, attack_type = MELEE_ATTACK)
+	var/obj/item/projectile/P = hitby
+	bullet_act(P)
+	return ..()
```
Comment (yoyobatty) [inline hippiestation/code/modules/clothing/spacesuits/hardsuit.dm:25]: FYI this override was necessary to get bullet_act to register, it doesn't work without this hit_reaction added.

**Comment by Maroonenahmoron** [comment]
How does it react to getting shot by multiple pellets at once, like a shotgun?

**Comment by yoyobatty** [comment]
Has a 15 second cooldown for all messages, should only display once.

**Comment by Maroonenahmoron** [comment]
I understand, seems a bit broken that way doesn't

**Comment by Zyzarda** [comment]
+1 but also add the noise when someone wearing a hardsuit dies.

**Comment by DaydreamPuppeh** [comment]
So if I get this straight, this is only adding a message and nothing else correct?

**Comment by yoyobatty** [comment]
Yes, if you wish to check the code, feel free.

**Comment by DaydreamPuppeh** [comment]
I would if I could. I don't understand how dream maker code works

**Comment by DaydreamPuppeh** [comment]
But in that case, +1

**Comment by Manly-Dwarf** [comment]
only if you merge my surplus pr

**Comment by yoyobatty** [comment]
I need to code check it, in the meantime just vote.

**Comment by cris5311** [comment]
+1

**Pull request merged**